app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -153,7 +153,7 @@ def parse_service(service)
       end
 
       new_result.merge!(
-        :portal_ip        => service.spec.portalIP,
+        :portal_ip        => service.spec.clusterIP,
```

Comment:
@oschreib not now, but please add a comment, so we'll re-evaluate (something like "TODO: in the future we may want to...")